ascify art/Ascify-Art.py

```diff
@@ -510,15 +539,16 @@ def add_more_fonts():
     l+=50
     if l>len(all_fonts)-50:
         l = len(all_fonts)
-        
+
 def loadcustom():
     # load custom font file
     global selected_font
-    font_open = filedialog.askopenfilename(filetypes =[('Font Files', ['*.ttf','*.otf','*.TTF'])])
-    if font_open:
+    if font_open := filedialog.askopenfilename(
+        filetypes=[('Font Files', ['*.ttf', '*.otf', '*.TTF'])]
+    ):

```

Comment:
Function `loadcustom` refactored with the following changes:

- Use named expression to simplify assignment and conditional